src/app/lab/lab.component.ts

```diff
@@ -89,9 +93,15 @@ export class LabComponent implements OnInit, OnDestroy {
             return;
         }
         this.removeChart();
-        this.indicator = indicator;
-        const chart = new Chart({indicator: indicator,
-                                 unit: indicator.default_units});
-        this.project.project_data.charts = [chart];
+        /*  Trigger lifecycle to truly destroy the chart component & its children
```

Comment:
I still dislike the decision to destroy/recreate, as completely destroying components and rebuilding is heavy handed and leads to cascading troublesome design decisions, such as the latest comment about removing the ngOnChanges no longer being necessary because we destroy/create. If we were ever to refactor and remove this destroy/create cycle, then the child ThresholdComponent would be broken because it no longer watches for the `@Input indicator` changes. As we continue to build based on this implicit behavior, it becomes easier to make similar decisions in the future which would further compound this issue.

With that said, I understand the impetus for it and would like to see the remaining tasks unblocked. A way to simplify this may become clear as we work through those tasks.